Kmozart/free-mentors
# #168260819 Remodelled UI

#### What does this PR do?
- Compares ```gh-pages``` with ```ch-remodelling-of-ui-168260189```
 
#### Description of Task to be completed?
- Merge ```gh-pages```  with ```ch-remodelling-of-ui-#168260819```

#### How should this be manually tested?
- Open ```https://kmozart.github.io/free-mentors/home.html``` and navigate through to confirm```

#### Any background context you want to provide?
- Had to redo the whole UI in order to get the right structure and design as required.

#### What are the relevant pivotal tracker stories?
-  #168260819  ch-remodelling-of-ui

#### Screenshots (if appropriate)
- N/A
#### Questions:
- N/A
